controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -152,8 +163,29 @@ func (r *SriovNetworkNodePolicyReconciler) Reconcile(ctx context.Context, req ct
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *SriovNetworkNodePolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	qHandler := func(q workqueue.RateLimitingInterface) {
+		q.AddAfter(reconcile.Request{NamespacedName: types.NamespacedName{
+			Namespace: namespace,
+			Name:      nodePolicySyncEventName,
+		}}, time.Second)
+	}
+
+	squashAndDebounceHandler := handler.Funcs{
+		CreateFunc: func(e event.CreateEvent, q workqueue.RateLimitingInterface) {
```

Comment:
Why we don't have DeleteFunc here?

Reply:
+1

a deleted policy also affects node states spec

Reply:
good point! fixing